README.md

```diff
@@ -3,15 +3,14 @@
 ![Nuget](https://img.shields.io/nuget/v/Quantoz.Nexus.Sdk.Token)
 
 Welcome to the Quantoz Nexus SDK repository. In here you will find all the tools required to make your connection with Nexus even easier!
-
```

Comment:
Please undo this

Reply:
Hi @nagarwal4  this is reverted https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/43/commits/b12c20b016bd0c228a064c3e64348b992ca2c932